tests/vmi_configuration_test.go

```diff
@@ -677,6 +694,27 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			})
 		})
 
+		Context("[Serial][rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with guest memory greater than requested memory and VFIO device present", Serial, func() {
```

Comment:
I think that the correct e2e test should be that we are able to use vm with VFIO devices if memory values are correct. The fail case is already tested by the unit test.